crates/sui-test-transaction-builder/src/lib.rs

```diff
@@ -51,19 +51,30 @@ impl TestTransactionBuilder {
     }
 
     pub fn move_call(
+        self,
+        package_id: ObjectID,
+        module: &'static str,
+        function: &'static str,
+        args: Vec<CallArg>,
+    ) -> Self {
+        self.move_call_with_type_args(package_id, module, function, args, vec![])
+    }
+
+    pub fn move_call_with_type_args(
```

Comment:
There is actually already a `with_type_args` function in the builder. Agreed it's very not obvious. Maybe a good idea to move that function up here to make it more discoverable.